<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolWindow</class>
 <widget class="QWidget" name="ToolWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tool:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Pen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eraser</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bucket</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="colorPreview">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>71</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setColor">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>71</y>
      <width>78</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
